src/_locales/hu/messages.json

```diff
@@ -0,0 +1,701 @@
+        "message": "Húzza a csúszkát balra egy tartomány tiltásához",
+        "description": ""
+    },
+    "options_enable_dnt_checkbox": {
+        "message": "A „<a href='https://globalprivacycontrol.org/' target='_blank'>globális adatvédelmi szabályozás</a>” és a „<a href='https://www.eff.org/issues/do-not-track' target='_blank'>ne kövessen</a>” jelzések küldése a weboldalaknak",
+        "description": "Checkbox label for enabling/disabling the Sec-GPC and DNT signals, found on the general settings page"
+    },
+    "options_disable_google_nav_error_service": {
+        "message": "Meglátogatott webcímek küldésének letiltása a Google számára. Ez letiltja a javaslatokat és a hasonló oldalakat, ha egy oldal nem található.",
+        "description": "Checkbox label found on the general settings page"
+    },
+    "options_disable_hyperlink_auditing": {
+        "message": "Hiperhivatkozás vizsgálatának letiltása",
+        "description": "Checkbox label found on the general settings page"
+    },
+    "options_disable_network_prediction": {
+        "message": "Előtöltés letiltása",
+        "description": "Checkbox label found on the general settings page"
+    },
+    "options_domain_filter_user": {
+        "message": "felhasználó által vezérelt",
+        "description": "Dropdown control setting on the Tracking Domains options page tab."
+    },
+    "popup_instructions": {
+        "message": "$COUNT$ lehetséges $LINK_START$követő$LINK_END$ tiltva",
+        "description": "Popup message shown when at least one tracker was blocked.",
+        "placeholders": {
+            "count": {
+                "content": "$1",
+                "example": "15"
+            },
+            "link_start": {
+                "content": "$2",
+                "example": "<a href='https://privacybadger.org/#What-is-a-third-party-tracker'>"
+            },
+            "link_end": {
+                "content": "</a>"
+            }
+        }
+    },
+    "popup_instructions_no_trackers": {
+        "message": "Nincsenek $LINK_START$követők$LINK_END$ tiltva",
+        "description": "Text shown in the popup when there are no trackers on the page.",
+        "placeholders": {
+            "link_start": {
+                "content": "$1",
+                "example": "<a href='https://privacybadger.org/#What-is-a-third-party-tracker'>"
+            },
+            "link_end": {
+                "content": "</a>"
+            }
+        }
+    },
+    "popup_blocked": {
+        "message": "Nincsenek harmadik féltől származó erőforrások ezen az oldalon. Éljen az adatvédelem!",
+        "description": "Text shown in the popup when showing non-tracking domains is enabled, and there are no third-party domains on the page."
+    },
+    "intro_by_eff": {
+        "message": "Az Electronic Frontier Foundation egy projektje",
+        "description": ""
+    },
+    "tooltip_allow": {
+        "message": "Húzza a csúszkát jobbra egy tartomány engedélyezéséhez",
```

Comment:
Does this say "drag" or "move"? It should say "move", as one can't actually drag Privacy Badger's sliders, only move/reposition them by clicking (or tapping).